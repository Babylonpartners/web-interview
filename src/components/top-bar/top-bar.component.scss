@import 'assets/styles/_constants.scss';

.top-bar {
  display: flex;
  position: sticky;
  top: 0;
  margin: 0 -12rem;
  justify-content: center;
  align-items: center;
  height: 60px;
  border-bottom: $border_standard;
  background-color: #fff;

  &__logo {
    height: 40px;

    img {
      height: 100%;
    }
  }
}
