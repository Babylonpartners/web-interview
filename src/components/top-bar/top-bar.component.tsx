import React, { FunctionComponent } from 'react'

import { Icon } from 'components/icon/icon.component'

import './top-bar.component.scss'

export const TopBar: FunctionComponent = () => {
  return (
    <div className="top-bar">
      <button className="top-bar__burger">
        <Icon alias="burger" />
      </button>
      <div className="top-bar__logo">
        <img src={require('assets/images/logo.png')} alt="Babylon is awesome" />
      </div>
    </div>
  )
}
