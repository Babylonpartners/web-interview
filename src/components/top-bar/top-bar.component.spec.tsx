import React from 'react'
import renderer from 'react-test-renderer'

import { TopBar } from './top-bar.component'

describe('TopBar', () => {
  it('should match snapshot', () => {
    const component = renderer.create(<TopBar />)

    expect(component.toJSON()).toMatchSnapshot()
  })
})
