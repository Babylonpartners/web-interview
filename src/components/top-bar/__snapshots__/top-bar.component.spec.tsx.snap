// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopBar should match snapshot 1`] = `
<div
  className="top-bar"
>
  <button
    className="top-bar__burger"
  >
    <span
      className="icon icon--burger"
      title="burger"
    />
  </button>
  <div
    className="top-bar__logo"
  >
    <img
      alt="Babylon is awesome"
      src="logo.png"
    />
  </div>
</div>
`;
